catalog/transformer/catalog-transformer-xml/src/main/java/ddf/catalog/transformer/xml/MetacardMarshallerImpl.java

```diff
@@ -257,14 +257,19 @@ private void writeAttributeToXml(PrintWriter writer, XmlPullParser parser, Attri
 
     private String geoToXml(BinaryContent content, XmlPullParser parser)
             throws UnsupportedEncodingException {
-        XppReader source = new XppReader(new InputStreamReader(content.getInputStream(),
-                StandardCharsets.UTF_8.name()), parser);
-
-        // if multi-threading, cannot abstract PrintWriter to class member
-        PrintWriter destination = writerProvider.build(Metacard.class);
-
-        new HierarchicalStreamCopier().copy(source, destination);
-
+        PrintWriter destination;
+        try (InputStreamReader inputStreamReader = new InputStreamReader(content.getInputStream(),
+                StandardCharsets.UTF_8.name())) {
+            XppReader source = new XppReader(inputStreamReader, parser);
+
+            // if multi-threading, cannot abstract PrintWriter to class member
+            destination = writerProvider.build(Metacard.class);
+
+            new HierarchicalStreamCopier().copy(source, destination);
+        } catch (IOException e) {
+            LOGGER.debug("Unable to marshall metacard from content");
+            return "";
```

Comment:
Almost certainly something will get jacked up down the line. It' probably better to throw a runtime exception.